samoa-api/pom.xml

```diff
@@ -101,6 +101,16 @@
       <version>${hadoop.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>0.8.2.1</version>
```

Comment:
We should factor the versions in a property (as it is done for other dependencies).